modules/auxiliary/scanner/http/splunk_web_login.rb

```diff
@@ -154,24 +154,30 @@ def do_login(user, pass)
           }
       })
 
-      if not res or res.code != 303
+      if not res
         vprint_error("FAILED LOGIN. '#{user}' : '#{pass}' with code #{res.code}")
```

Comment:
Might as well modernize the if logic too (use unless or ! etc)

> On Feb 18, 2015, at 00:37, James Lee <notifications@github.com> wrote:
> 
> In modules/auxiliary/scanner/http/splunk_web_login.rb:
> 
> > @@ -154,24 +154,30 @@ def do_login(user, pass)
> >            }
> >        })
> >  
> > -      if not res or res.code != 303
> > +      if not res
> >          vprint_error("FAILED LOGIN. '#{user}' : '#{pass}' with code #{res.code}")
> This will blow up because res is nil
> 
> —
> Reply to this email directly or view it on GitHub.
>